scripts/build_bus_regions.py

```diff
@@ -189,6 +220,13 @@ def custom_voronoi_partition_pts(points,
         # print(shapely.validation.explain_validity(onshore_shape), onshore_shape.area)
         onshore_locs = n.buses.loc[c_b & n.buses.substation_lv, ["x", "y"]]
         # print(onshore_locs.values)
+        if snakemake.config['alternative_clustering']:
```

Comment:
The approach seems to be:
- if alternative_clustering is in place: then (a) set bus_regions shapes using gadm, instead of voronoi, (b) re-process the results in build_renewable_profiles and (c) perform the clustering with the shapes.
- otherwise, do the typical workflow

I was wondering that maybe the (a-c) may not need to be cross-linked, they may be two separate options:
- in bus_regions: use voronoi or gadm shapes; when gadm shapes are processed, the inputs may be prepared to avoid the special option in build_renewable profiles maybe
- in cluster_network we provide an optional shape that is used to clusterize. When gadm option is used in bus_regions and the same shape is used in cluster_network we shall obtain the same result of the current process, but the user is not constrained to use both at the same time only. What do you think?
To me the above may be simple to implement; yet, we could decide whether to do it in this pull request or later.

Reply:
I like the suggestion, and I think it will be useful in some use cases. I can separate it into 2 different options, but I think it can also be added at a later stage.